.github/pull_request_template.md

```diff
@@ -6,7 +6,7 @@ Please title your PR according to the following types and scopes following [conv
 - `chore(<type>):` will not trigger any release and should be used for internal repo changes
 - `<type>(public):` will trigger a patch version for non-code changes (e.g. README changes)
 - `feat(SDK name):` will trigger a minor version
-- `feat(!):` will trigger a major version for a breaking change
+- `feat(breaking):` will trigger a major version for a breaking change
```

Comment:
does this scope to a specific SDK? if so how

ie can I do `feat(breaking sdk-core)`?

Reply:
The scope will need to be `breaking` literal, but which SDK it affects will only be based on the files changed! So changing for `sdk-core` would mean only touching files in that SDK for the PR with that title and `semantic-release` will handle automatically!